framework/Driver.py

```diff
@@ -167,6 +167,7 @@ def checkVersions():
 
   if interfaceCheck:
     os.environ['RAVENinterfaceCheck'] = 'True'
+    print('Interface CHECK activated!\n')
```

Comment:
why do we use print statement and not the RAVEN messaging standard?

Reply:
Because at this stage the message handler is not initialized yet. There is the plan to modify how it works and make it static in the near future.